cirq-pasqal/cirq_pasqal/pasqal_device.py

```diff
@@ -34,6 +34,22 @@ class PasqalDevice(cirq.devices.Device):
     execution on the specified device are handled internally by Pasqal.
     """
 
+    gateset = cirq.Gateset(
```

Comment:
Nit: Class attributes and globals are discouraged, can we move this to `__init__` ?

Reply:
done